fe/topology.py

```diff
@@ -470,6 +469,54 @@ def interpolate_params(self, params_a, params_b):
 
         return jnp.array(src_params), jnp.array(dst_params)
 
+    def interpolate_nonbonded_params(self, params_a, params_b):
+        """
+        Special interpolation method for nonbonded parameters. For R-group atoms,
+        their charges and vdw eps parameters are scaled to zero. Vdw sigma
+        remains unchanged. This method is needed in order to ensure that R-groups
+        that branch from multiple distinct attachment points are fully non-interacting
```

Comment:
Sorry, I'm having trouble following the current description of the problem in this docstring. It's unclear to me why the issue would only arise when considering multiple R-groups that branch from multiple distinct attachment points, but not when a single R group is large enough to contain atom pairs that interact via nonbonded terms. Perhaps it would be clarifying to add even a bit more description of the desired behavior here-- or add a pointer to the behavior of a failing test case when the issue is not handled properly.

Reply:
I'm actually not 100% sure for the case of a large single R-group, I know that with Boresch restraints for ABFE, we do not need to turn off the ligand's nonbonded interactions, even though they branch off from a single virtual bond. I think it's always safer to turn them off than to leave them on, and it's more consistent with the behavior seen in perses.